lib/yt/errors/forbidden.rb

```diff
@@ -0,0 +1,12 @@
+require 'yt/errors/request_error'
+
+module Yt
+  module Errors
+    class Forbidden < RequestError
+      # @ return [String] to give explanation of the error
```

Comment:
@kangkyu I would remove this description. It's not very helpful. It's the same as the name of the method.